tox.ini

```diff
@@ -93,3 +93,10 @@ deps=pidiff
 skip_install=true
 commands=pidiff pubtools-iib .
 
+[testenv:py3-bandit-exitzero]
+deps = bandit
+commands = bandit -r . -l --exclude './.tox' --exit-zero
```

Comment:
Shouldn't this be '-ll' instead of '-l' ?

Reply:
This instance of the Bandit run (exitzero) is intentionally set to '-l'. 

It will list all low and higher findings in a non-failing fashion in the CI log, so they can be corrected if desired.

There is another run below which only contains medium and higher findings, which will fail if any issues are found.